instagram/models.py

```diff
@@ -81,10 +81,10 @@ def object_from_dictionary(cls, entry):
         for version, version_info in six.iteritems(entry['images']):
             new_media.images[version] = Image.object_from_dictionary(version_info)
 
-        if new_media.type == 'video':
-            new_media.videos = {}
-            for version, version_info in six.iteritems(entry['videos']):
-                new_media.videos[version] = Video.object_from_dictionary(version_info)
+        # if new_media.type == 'video':
+        #     new_media.videos = {}
+        #     for version, version_info in six.iteritems(entry['videos']):
```

Comment:
since the `videos` key no longer appears to exist, I imagined that accessing the dict's key using `.get()` instead of array notation would resolve this, but it did not. Had to resort to commenting it out entirely.